graspy/inference/latent_distribution_test.py

```diff
@@ -43,86 +45,59 @@ class LatentDistributionTest(BaseInference):
 
     Attributes
     ----------
+    null_distribution_ : np.ndarray
+        The distribution of T statistics generated under the null.
+
     sample_T_statistic_ : float
-        The observed difference between the embedded latent positions of the two 
-        input graphs.
+        The observed difference between the embedded positions of the two input graphs
+        after an alignment (the type of alignment depends on `test_case`)
 
     p_ : float
         The overall p value from the test.
-    
-    null_distribution_ : ndarray, shape (n_bootstraps, )
-        The distribution of T statistics generated under the null.
 
-    References
-    ----------
-    .. [2] Tang, M., Athreya, A., Sussman, D. L., Lyzinski, V., & Priebe, C. E. (2017). 
-        "A nonparametric two-sample hypothesis testing problem for random graphs."
-        Bernoulli, 23(3), 1599-1630.
+    Examples
+    --------
+    >>> npt = LatentDistributionTest(n_components=2, which_test='mgc')
+    >>> p = npt.fit(A1, A2)
+
+    See also
+    --------
+    graspy.embed.AdjacencySpectralEmbed
+    graspy.embed.OmnibusEmbed
+    graspy.embed.selectSVD
     """
+    # TODO: reference Varjavand paper when it is on arxiv
 
-    def __init__(self, n_components=None, n_bootstraps=200, bandwidth=None):
+    def __init__(self, n_components=None, which_test="mgc", graph=False):
         if n_components is not None:
```

Comment:
can this be in `base`?

Reply:
yes!

Reply:
I think I meant that `n_components`, `n_bootstraps` are both in LPT and so the checks for them, and saving them as class attributes, can probably be in base. LMK if you disagree